### ISSUE TITLE
No bundle in fresh install

### ISSUE THREAD

**Issue by zamith** [rails/rails-dev-box#84]
Just installed this with a `vagrant up` and it does not have bundler.

I get the following error:

```
The program 'bundle' is currently not installed. To run 'bundle' please ask your administrator to install the package 'bundler'
```

If I try to install it, here's what happens:

```
$ gem install bundler
ERROR:  Could not find a valid gem 'bundler' (>= 0) in any repository
ERROR:  Possible alternatives: bundler
```

**Comment by nruth** [comment]
Please excuse me raising the dead, but in case someone else ends up here through Google, try the following from [ask ubuntu](http://askubuntu.com/questions/238040/how-do-i-fix-name-service-for-vagrant-client/296796#296796) 

```ruby
config.vm.provider "virtualbox" do |vb|
  vb.customize ['modifyvm', :id, "--natdnshostresolver1", "on"]
end
```

It was breaking ubuntu 14 (but not 12) when trying to add the brightbox-ng ppa on my Mac.

**Comment by robcaw** [comment]
@nruth :+1: Thank you!
